Author: Jasper Grimm
Use the JAVAFX_HOME environment variable instead of hardcoding the JavaFX path
--- jloda.orig/antbuild/build.xml	2021-09-02 12:55:48.000000000 +0100
+++ jloda/antbuild/build.xml	2021-09-02 12:56:57.000000000 +0100
@@ -21,7 +21,8 @@
 	        <property name="ant.build.javac.source" value="13"/>
         	<property name="ant.build.javac.target" value="13"/>
 
-		<property name="jfxDir" value="../../javafx/lib"/>
+                <property environment="env"/>
+		<property name="jfxDir" value="{env.JAVAFX_HOME}/lib"/>
 		<property name="buildSrcDir" value="../antbuild/src"/>
 		<property name="buildModulesDir" value="../antbuild/modules"/>
 
